marketpulse/main/urls.py

```diff
@@ -8,4 +8,5 @@
     url(r'^activities/$', 'activities', name='activities'),
     url(r'^manifest\.webapp', 'manifest', name='manifest'),
     url(r'^fxosprice/', include('{0}.main.fxosprice_urls'.format(settings.PROJECT_MODULE))),
+    url(r'^fxosmedia/', include('{0}.main.fxosmedia_urls'.format(settings.PROJECT_MODULE))),
```

Comment:
Maybe we should use something more generic like `/media/` or `/upload`. Thoughts?